java-vtl-script/src/main/java/no/ssb/vtl/script/visitors/functions/NativeFunctionsVisitor.java

```diff
@@ -65,41 +61,15 @@ public NativeFunctionsVisitor(VTLBaseVisitor<VTLExpression2> expressionVisitor)
     @Override
     public VTLExpression2 visitNativeCall(VTLParser.NativeCallContext ctx) {
         if (functions.containsKey(ctx.functionName.getText())) {
+            VTLFunction<VTLObject> functionInstance = functions.get(ctx.functionName.getText());
```

Comment:
This is pretty cool.
Would [`ServiceLoader`](http://docs.oracle.com/javase/8/docs/api/java/util/ServiceLoader.html) work to load all implementations of native functions?

Reply:
I don't understand?